src/coreclr/src/jit/lower.cpp

```diff
@@ -3057,7 +3057,12 @@ void Lowering::LowerRetStruct(GenTreeUnOp* ret)
             LowerRetStructLclVar(ret);
             break;
 
+#ifdef FEATURE_SIMD
         case GT_SIMD:
+#endif // FEATURE_SIMD
+#ifdef FEATURE_HW_INTRINSICS
+        case GT_HWINTRINSIC:
```

Comment:
Was this a case introduced by the move of some `GT_SIMD` intrinsics to be `GT_HWINTRINSIC`?

(Just want to understand if its a case I missed and where the error was raised so I can check it next time... 😄)

Reply:
Yes, but it was happening only with `JitOldStructRetyping=0` that is disabled by default.